init.lua

```diff
@@ -30,6 +30,20 @@ local function destroy(handle)
 end
 ffi.gc(cudnn.handle, destroy)
 
+cudnn.typemap = {
+   ['torch.CudaHalfTensor']   = 'CUDNN_DATA_HALF',
+   ['torch.CudaTensor']       = 'CUDNN_DATA_FLOAT',
+   ['torch.CudaDoubleTensor'] = 'CUDNN_DATA_DOUBLE',
+}
+
+-- TODO: determine if device supports true half and use true half on it
```

Comment:
To determine that, we probably should asume all devices are the same, or can we map those per device?
It should be fair assumption though, and the check should be: 
true if   a) we are running  compute_60 code or above
       or  b) 'GPU device is TX1 or Tegra X1' 
Do we currently have runtime info for such check ?
Instead of b) we may check if the CPU is ARM (I think).